src/containers/PageBuilder/SectionBuilder/SectionContainer/SectionContainer.module.css

```diff
@@ -23,6 +23,8 @@
   padding: 32px 0;
   position: relative;
   word-break: break-word;
+  width: 100%;
+  /* also, TODO, see if this breaks anyting */
```

Comment:
This is a good question - since this affects every section type. Is this needed at all?

Another option would be to create some way for a sectionSectionBuilder to pass forward a prop or class that overrides or extends .sectionContent class.
- https://github.com/sharetribe/web-template/blob/main/src/containers/PageBuilder/SectionBuilder/SectionContainer/SectionContainer.js#L45
- https://github.com/sharetribe/web-template/blob/main/src/containers/PageBuilder/SectionBuilder/SectionBuilder.js#L129

That would allow passing a property that's only used if the _search CTA_ is in use.

However, I'm not sure if the extra complexity makes sense. The `width: 100%` is more straightforward solution.